cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/scheduler/SchedulerQueueResolver.java

```diff
@@ -34,9 +35,7 @@
   private final String defaultQueue;
   private final NamespaceStore store;
 
-  /**
-   * Construct SchedulerQueueResolver with CConfiguration and Store.
```

Comment:
You don't like javadoc? Also, make the constructor package private (i.e. remove the `public`).